setup.py

```diff
@@ -52,20 +52,17 @@ def get_version():
     packages=packages,
     package_data={'': ['LICENSE', 'AUTHORS.rst']},
     include_package_data=True,
-    python_requires='>=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*',
+    python_requires='>=3.7, <3.11',
```

Comment:
Don't add an upper limit:

```suggestion
    python_requires='>=3.7',
```